packages/lesswrong/components/posts/PostsItem2.tsx

```diff
@@ -317,9 +339,13 @@ const PostsItem2 = ({
   // dismissRecommendation: If this is a Resume Reading suggestion, a callback
   // to dismiss it.
   dismissRecommendation,
+  draft,
+  // toggleDeleteDraft, if this a draft, a callback to archive/unarchive it
```

Comment:
```suggestion
  // toggleDeleteDraft, if this a draft, a callback to archive/unarchive it
```
You're not the progenitor of this problem in this file, but docstrings for arguments should go on the type, see below.